packages/vscode-extension/src/resources/Strings.json

```diff
@@ -197,7 +197,8 @@
       "emptyData": "Empty zip file",
       "downloadSampleFail": "Failed to download sample app",
       "quickStartPageTitle": "Quick Start",
-      "samplePageTitle": "Samples"
+      "samplePageTitle": "Samples",
+      "surveyPageTitle": "TeamsFx Survey"
```

Comment:
Is 'Teamsfx' something we want to expose to user?

Reply:
good catch